device-types/Quantum/Scalar-i3.yaml

```diff
@@ -0,0 +1,29 @@
+---
+manufacturer: Quantum
+model: Scalar i3
+slug: quantum-scalar-i3
+description: LTO library for long-term data archiving
+u_height: 3.0
+is_full_depth: true
```

Comment:
Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

Reply:
Of course, but unfortunately I could not find any mention of these into [the official datasheet](https://cdn.allbound.com/iq-ab/2021/11/Scalar-Tape-DS00559A.pdf). And ours is already in production, so I cannot weight it anymore.